docs/support_scripts/create_json_responses/app.py

```diff
@@ -272,17 +251,21 @@ def get_endpoint_final(entity_id, endpoint, csv):
     '["riskScore"]',
 ]
 
-csvs = ["crec"]
+csvs = ["bonds"]
 
 search_headers = [
     "endpoint",
     "entity_ref",
-    "search_entity",
-    "search_field",
     "full_check",
 ]
 
+print(environment)
```

Comment:
Probably best to remove this or you'll be leaking secrets all over the shop.

Reply:
This is just a little helper script that I use to generate the shape of the API data to test manually. It doesn't get run anywhere as part of our pipelines. 

There's loads of print statements all over the place here that would print sensitive stuff if it were used in a non local env but as I say, it doesn't get run as part of the pipeline and is only a helpful script for dev purposes :)